taxi/models.py

```diff
@@ -0,0 +1,32 @@
+from tkinter.constants import CASCADE
```

Comment:
The import `from tkinter.constants import CASCADE` is unnecessary and should be removed. The `CASCADE` used in the `ForeignKey` is from `django.db.models`, not `tkinter.constants`.